APCAppCore/APCAppCore/UI/Onboarding/APCStudyOverviewCollectionViewController.m

```diff
@@ -238,7 +238,8 @@ - (UICollectionViewCell *)collectionView:(UICollectionView *)collectionView cell
         
         NSString *filePath = [[NSBundle mainBundle] pathForResource: studyDetails.detailText ofType:@"html" inDirectory:@"HTMLContent"];
         NSAssert(filePath, @"Expecting file \"%@.html\" to be present in the \"HTMLContent\" directory, but didn't find it", studyDetails.detailText);
-        NSURL *targetURL = [NSURL URLWithString:filePath];
+        NSString *encodingFilePath = [filePath stringByAddingPercentEscapesUsingEncoding:NSUTF8StringEncoding];
+        NSURL *targetURL = [NSURL URLWithString:encodingFilePath];
```

Comment:
yes, for example: if the Bundle Identifier contain spaces or special characters surely will not Appear the HTML in the WebView, because the NSURL can not be built, it Shall be made a null NSURL. So is you need to use Encoding.

=) Use it:
 NSURL *targetURL = [NSURL fileURLWithPath: filePath];